packages/terra-functional-testing/src/reporters/fileOutput-reporter/index.js

```diff
@@ -0,0 +1,211 @@
+const fs = require('fs-extra');
+const path = require('path');
+const stripAnsi = require('strip-ansi');
+const SpecReporter = require('@wdio/spec-reporter').default;
+const endOfLine = require('os').EOL;
+const { Logger } = require('@cerner/terra-cli');
+
+const LOG_CONTEXT = '[Terra-Toolkit:fileOutput-reporter]';
```

Comment:
Most of our folders are named in snake case. We should avoid using both snake case with camel case.

Reply:
https://github.com/cerner/terra-toolkit/pull/665/files#diff-d8f1b66e8db21668273f6ad5ba417c4a5af04b1fcd0e46b5e2923ac1345fa743R1